src/editor_view.cpp

```diff
@@ -244,22 +244,7 @@ void EditorView::mouseMoveEvent(QMouseEvent * event)
             if (m_mediator.selectionGroupSize()) {
                 m_mediator.moveSelectionGroup(*node, m_grid.snapToGrid(m_mappedPos - m_mediator.mouseAction().sourcePosOnNode()));
             } else {
-                auto cur_pos = m_mappedPos - m_mediator.mouseAction().sourcePosOnNode();
```

Comment:
Ok, ignore my comments I added on this removed block. These two commits could be squashed together.